### PR TITLE
A little bit of cleanup and reorganization

### PR DESCRIPTION
**Pull request by cjihrig**

### CODE REVIEW

**Comment by jaw187** [comment]
I would merge this, but @backer said he was going to "reek havoc" on this code yesterday.

File: hapchat/public/photocap.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootswatch/3.2.0/united/bootstrap.min.css">
-    <link rel="stylesheet" href="/static/css/main.css">
+    <link rel="stylesheet" href="/css/main.css">
```
Comment (arb) [inline hapchat/public/photocap.html]: Please change back to /static/, without, we can't have a "catch-all" get route to redirect people back to the home page. Plus this is how I see it most of the time when not using Express.

File: hapchat/public/photocap.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <a class="navbar-brand" href="#"><img src="/static/logo.png" height="24" /> Hapchat</a>
+            <a class="navbar-brand" href="#"><img src="/logo.png" height="24" /> Hapchat</a>
```
Comment (arb) [inline hapchat/public/photocap.html]: Same as above.

File: hapchat/public/photocap.html [hunk 3]
```diff
@@ -243,6 +243,6 @@ <h4 class="modal-title">Photo Taker</h4>
 
 <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
 <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
-<script src="/static/js/main.js"></script>
+<script src="/js/main.js"></script>
```
Comment (arb) [inline hapchat/public/photocap.html]: Same as above.

**Pull request merged**